eaf_pyqterm_backend.py

```diff
@@ -48,25 +48,25 @@ def _spawn_pty(self, env, argv, start_directory):
 
     def _spawn_winpty(self, env, argv, start_directory):
         self.pty = pty.spawn(
-            list(argv[0]) + argv,
+            argv,
             start_directory,
             env,
-            (env["COLUMNS"], env["LINES"]),
+            (int(env["COLUMNS"]), int(env["LINES"])),
         )
 
     def read(self):
         return self.pty.read(65536)
 
     def write(self, data):
-        self.pty.write(data)
+        self.pty.write(data.decode())
```

Comment:
It's very wrong, if some code can't work on Windows, you should use `platform.system()` add branch.

This change will make terminal hiden when we type any char on Linux.

I have fixed it with patch https://github.com/mumu-lhl/eaf-pyqterminal/pull/38